src/ngraph/ngraph_compiler.cc

```diff
@@ -459,6 +459,10 @@ void Compiler::DeepCopy(const nnvm::Graph& graph) {
 }
 
 bool bad_type(const NodePtr& node) {
+  if (!ngraph_context_fallback() &&
```

Comment:
we need a test for this functionality, were you able to check by exporting env.

Reply:
Hmm, getting an error if I export the env variable, doesn't make much sense, will debug.

Reply:
Fixed.